### ISSUE TITLE
Windows Hi DPI support requires a settings change by the user?

### ISSUE THREAD

**Issue by jasonsanjose** [adobe/brackets#10178]
I'm running 1.1.0-15542 on Win 8 on a Samsing Ativ Book 9 (3200 x 1800) with the display set at 200%. On the left is Brackets 1.1 out of the box, on the right is Brackets with the "Disable display scaling on high DPI settings" checkbox enabled.

Notice with the setting disabled...
* the titlebar minimize/maximize/exit icons are teeny tiny
* the in-browser fonts and menu font is crisper

![windows hi dpi](https://cloud.githubusercontent.com/assets/1148713/5444109/b0d40c7a-845c-11e4-9867-1e3e1b8de54f.png)

Do we expect users to do this manually or should it be automatically detected? References:

http://www.magpcss.org/ceforum/viewtopic.php?f=6&t=12366
http://msdn.microsoft.com/en-us/library/windows/desktop/dn469266%28v=vs.85%29.aspx
https://code.google.com/p/chromiumembedded/issues/detail?id=1445

**Comment by pthiess** [comment]
@ryanstewart We need your input on this team nomination.

**Comment by peterflynn** [comment]
In review last week we talked about splitting this into two pieces:

1. Change the manifest only, so the setting change isn't required.
2. Update the custom window chrome so the minimize/maximize/restore icons are drawn & position at the correct size (instead of tiny as in screenshot above).

The benefit seems big enough that it might be worth getting part 1 merged sooner -- the idea is that people would be ok living with the tiny window-size icons until part 2 lands later, in exchange for getting automatic high DPI support sooner.  @JeffryBooher I think you were going to post an estimate of how much work it is do to just the manifest change?

**Comment by JeffryBooher** [comment]
@peterflynn sorry, I thought I mentioned that this work would take only a day to add and test during the meeting.  Either we add a manifest file since there isn't one and add the setting

```xml
<assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0" xmlns:asmv3="urn:schemas-microsoft-com:asm.v3" >
  <asmv3:application>
    <asmv3:windowsSettings xmlns="http://schemas.microsoft.com/SMI/2005/WindowsSettings">
      <dpiAware>true</dpiAware>
    </asmv3:windowsSettings>
  </asmv3:application>
</assembly>
```

or we can just call `SetProcessDpiAwareness(true);` on startup -- which seems easier since we don't need to worry about adding a manifest file to the repo and that change would require changing the grunt script as well...

**Comment by peterflynn** [comment]
@JeffryBooher Oh cool, ok!  I'm going to nominate this for 1.2 then, since it seems relatively manageable to get that first half in...

**Comment by nethip** [comment]
Initial implementation done. Needs rigorous testing on all platforms. Moving this to 1.3

**Comment by jasonsanjose** [comment]
@nethip It looks like this landed in the 1.2 release branch in brackets-shell. Is that what you intended?

**Comment by nethip** [comment]
@jasonsanjose Yes indeed :smile: I am closing this bug.

**Comment by MarcelGerber** [comment]
@nethip @jasonsanjose Either one of you should set the milestone of this issue back to 1.2 if it gets shipped with 1.2

**Comment by nethip** [comment]
@MarcelGerber Thanks! Changed it to Release 1.2
